src/payment/strategies/stripev3/stripev3-payment-strategy.ts

```diff
@@ -108,7 +111,7 @@ export default class StripeV3PaymentStrategy implements PaymentStrategy {
         try {
             return await this._store.dispatch(this._paymentActionCreator.submitPayment(paymentPayload));
         } catch (error) {
-            return await this._processAdditionalAction(error, methodId, shouldSaveInstrument, shouldSetAsDefaultInstrument);
+            return await this._processAdditionalAction(error, stripeError, methodId, shouldSaveInstrument, shouldSetAsDefaultInstrument);
```

Comment:
I personally don't like to keep adding arguments to this function, in fact, `processAdditionalAction` was intended for only for that matter. 

In this case what I would do is just create an `if` block and validate `error`